client/galaxy/style/scss/ui/search-input.scss

```diff
@@ -29,10 +29,10 @@
         font-size: $font-size-base;
         line-height: $line-height-base;
         color: $text-color;
-        border: $border-default;
+        border: none;
         @include border-radius($border-radius-extralarge);
         max-width: auto;
-        background: $white;
+        background: $brand-secondary;
```

Comment:
This background is too dark, it makes the search field look like a button, also all other text fields have bright colors given the existing blue.css theme configuration. Thanks again for working on this and thank you for your contribution.

Reply:
This background was chosen so it did not need a border, and to stay within the same color palette that is now being used. But I agree, it is now more like a button. I had a feeling there was on purpose a distinction made between the two search fields, the ones that are very rounded and the more rectangular ones.